ci-operator/config/openshift/cincinnati/OWNERS

```diff
@@ -1,10 +1,11 @@
 # DO NOT EDIT; this file is auto-generated using tools/populate-owners.
-# from https://github.com/openshift/cincinnati/blob/760d5add6066735e38c28a294ee1446ce5eab215/OWNERS
+# from https://github.com/openshift/cincinnati/blob/master/OWNERS when master is at 1e38099d1e5ed884eec5c83c9f9716b1fa8fb2b7
 # See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
 
 approvers:
 - crawford
-- steveeJ
 - lucab
-- wking
+- steveej
```

Comment:
comming from `github.NormLogin(alias)` function, we could disable this if we want.

Reply:
Find the answer. [By definition](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md), alias and names are case insensitive.